content/blog/2021/01/camel-quarkus-release-1.6.0/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Camel Quarkus 1.6.0 Released"
+date: 2021-01-18
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.6.0 Released"
+summary: "The highlights of Camel Quarkus 1.6.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+We are pleased to announce the release 1.6.0 of Camel Quarkus! Here are the highlights.
+
+## Java 11 required
+
+Having deprecated Java 8 [long time ago](https://camel.apache.org/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/) and
+following the same requirement by Quarkus, Camel Quarkus now requires Java 11+.
+
+## Support for more Camel components
+
+Camel Quarkus brings support for three new Camel bits:
```

Comment:
```suggestion
Camel Quarkus brings support for three new Camel components:
```

Reply:
Some time ago, I started using "bits" when I speak about languages, data formats and "others" in addition to components. Does "bits" sound too strange? I do not mind changing it here, but would still be nice to hear your your native speaker's opinion about "bits" :)

Reply:
Replaced in 857ef0c

Reply:
'bits' is like a casual, catch-all expression that can mean pretty much anything you like. So it doesn't really describe what three new things we have added support for.

As ever, I'm pretty relaxed about the language. So it's your choice whether to change it or not.